src/main/java/uk/ac/ebi/eva/commons/models/data/VariantSourceEntity.java

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright 2016 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.commons.models.data;
+
+import org.opencb.biodata.models.variant.VariantSource;
+import org.opencb.biodata.models.variant.VariantStudy;
+import org.opencb.biodata.models.variant.stats.VariantGlobalStats;
+import org.springframework.data.mongodb.core.mapping.Document;
+import org.springframework.data.mongodb.core.mapping.Field;
+
+import java.util.Calendar;
+import java.util.Date;
+import java.util.Map;
+
+/**
+ * Represents a file (VariantSource) in a database.
+ * <p>
+ * TODO jmmut: VariantSource also has aggregation and pedigree
+ */
+@Document
+public class VariantSourceEntity {
```

Comment:
now you say it, I thought it should, but it doesn't really create anything according to my tests. It might be because the other Document had indexes.